bindings/c/CMakeLists.txt

```diff
@@ -347,84 +347,84 @@ if (0) # reenable after stabilizing the test
-
-    add_test(NAME fdb_c_wiggle_and_upgrade_latest
-      COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
-          --build-dir ${CMAKE_BINARY_DIR}
-          --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-          --upgrade-path "7.1.9" "wiggle" "7.2.0"
-          --disable-log-dump
-          --process-number 3
-          --redundancy double
-        )
-
-      add_test(NAME fdb_c_wiggle_and_upgrade_63
-        COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
-            --build-dir ${CMAKE_BINARY_DIR}
-            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
-            --upgrade-path "6.3.24" "wiggle" "7.0.0"
-            --disable-log-dump
-            --process-number 3
-            --redundancy double
-          )
-
-  endif()
+#  if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND NOT USE_SANITIZER)
+#    add_test(NAME fdb_c_upgrade_single_threaded_630api
+#      COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
+#            --build-dir ${CMAKE_BINARY_DIR}
+#            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadSingleThr.toml
+#            --upgrade-path "6.3.23" "7.0.0" "7.1.9" "7.2.0"
+#            --process-number 1
+#            )
+#
+#    add_test(NAME fdb_c_upgrade_single_threaded_700api
+#      COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
+#            --build-dir ${CMAKE_BINARY_DIR}
+#            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadSingleThr.toml
+#            --upgrade-path "7.0.0" "7.1.9" "7.2.0"
+#            --process-number 1
+#            )
+#
+#    add_test(NAME fdb_c_upgrade_multi_threaded_630api
+#        COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
+#            --build-dir ${CMAKE_BINARY_DIR}
+#            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
+#            --upgrade-path "6.3.23" "7.0.0" "7.1.9" "7.2.0" "7.1.9"
+#            --process-number 3
+#          )
+#
+#    add_test(NAME fdb_c_upgrade_multi_threaded_700api
+#        COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
+#            --build-dir ${CMAKE_BINARY_DIR}
+#            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
+#            --upgrade-path "7.0.0" "7.1.9" "7.2.0" "7.1.9"
+#            --process-number 3
+#          )
+#
+#    add_test(NAME fdb_c_upgrade_multi_threaded_710api
+#        COMMAND ${CMAKE_SOURCE_DIR}/tests/TestRunner/upgrade_test.py
+#            --build-dir ${CMAKE_BINARY_DIR}
+#            --test-file ${CMAKE_SOURCE_DIR}/bindings/c/test/apitester/tests/upgrade/MixedApiWorkloadMultiThr.toml
+#            --upgrade-path "7.1.9" "7.2.0" "7.1.9"
+#            --process-number 3
+#          )
+#
+#    add_test(NAME fdb_c_cluster_wiggle
```

Comment:
At least that one should work

Reply:
Sounds good, i'll re-enable the ones that should work in a later pr